docs/includes/global-attributes.asciidoc

```diff
@@ -10,13 +10,24 @@
     License: GFDL 1.2 or later. http://www.gnu.org/licenses/fdl.html
 
 ////
-
-// TODO have the build system take care of this
-
 :release_version: 10.2.0
+:release_month: October
+:encoding: UTF-8
+:lang: en
+:doctype: book
+PacketFence v{release_version}
```

Comment:
I'm not sure it's possible to add text directly here. I know it works but it seems it can cause unexpected behaviors.

On Network guide, it hide all other attributes declared after text.

I suggest to add this text directly after General title (in main AsciiDoc file) with `:release_version:`attribute declared **before** if we want to keep same behavior or include `global-attributes.asciidoc` before text.

Reply:
Second solution doesn't work.

Reply:
What are the _unexpected behaviors_?

Reply:
On Network Guide, due to a blank line after main title, reading of `docs/includes/global-attributes.asciidoc` was stopped after `PacketFence v{release_version}`. It means that `images` directory was unset and images were not found.

Reply:
How about fixing the Network Guide instead?

Reply:
Let's go with this solution. It's already done.